src/JustEat.StatsD/PooledUdpTransport.cs

```diff
@@ -10,10 +10,10 @@ namespace JustEat.StatsD
     /// </summary>
     public sealed class PooledUdpTransport : IStatsDTransport, IDisposable
     {
-        private const int PoolSize = 30;
+        private const int InitialPoolSize = 2;
```

Comment:
Maybe we could do this (or something similar) instead? Kestrel does something similar to determine the default size of its worker thread pool. Then in theory we have enough pre-allocated for all cores to be busy doing one thing simultaneously.

```csharp
private static int InitialPoolSize = Environment.ProcessorCount * 2;
```